products/views.py

```diff
@@ -1,7 +1,28 @@
-<<<<<<< HEAD
-=======
+from django.views    import View
+from django.http     import JsonResponse
 
+from products.models import Product, Category, SubCategory, Allergy, ProductInformation, AllergyProduct, Image, DiscountRate
```

Comment:
```suggestion
from products.models import (
    Product, Category, 
    SubCategory, Allergy, 
    ProductInformation, 
    AllergyProduct, Image, 
    DiscountRate
)
```
임포트가 많아지면 다음과 같이 해봅시다. 만약 이것보다 더 많이지면 한줄당 3개 혹은 네개씩 선언해주세요.

Reply:
수정 했습니다~